schedule.json

```diff
@@ -32,11 +32,11 @@
     "end": "2017-06-30"
   },
   "v8": {
-    "start": "2017-04-30",
+    "start": "2017-05-30",
     "lts": "2017-10-31",
-    "maintenance": "2019-04-01",
+    "maintenance": "2019-05-01",
```

Comment:
I thought I saw that somewhere but I can't find anything saying that so I am going to change it back.